package.json

```diff
@@ -11,11 +11,10 @@
     "@chakra-ui/core": "^0.8.0",
     "bootstrap": "^5.1.3",
     "css-loader": "^6.5.1",
-    "firebase": "^9.6.2",
+    "firebase": "^8.10.0",
```

Comment:
The new changes in package.json show "firebase": "^8.10.0",  we need it to be 9.6.2.

Reply:
I am not sure where this 8.10 comes from... As I have "firebase": "^9.6.2",

Reply:
I will push this merge and then can you push again with the 9.6.2. ?